android/libraries/rib-base/src/test/java/com/badoo/ribs/core/RouterTest.kt

```diff
@@ -17,56 +19,56 @@ class RouterTest {
     private lateinit var routingActionForC3: RoutingAction<TestView>
     private lateinit var routingActionForC4: RoutingAction<TestView>
     private lateinit var routingActionForC5: RoutingAction<TestView>
-    private lateinit var permanentPartBuilder1: () -> Node<*>
-    private lateinit var permanentPartBuilder2: () -> Node<*>
     private lateinit var node: Node<TestView>
-    private lateinit var childNode1: Node<*>
-    private lateinit var childNode2: Node<*>
-
-    companion object {
-        private val initialConfigurationIsC2 = TestRouter.Configuration.C2
-    }
+    private lateinit var childNodeC2_1: Node<*>
+    private lateinit var childNodeC2_2: Node<*>
 
     @Before
     fun setUp() {
+        Thread.currentThread().setUncaughtExceptionHandler { _, _ -> assert(false) }
```

Comment:
why do we need this?

Reply:
We met a curious case of RxJava errors not failing the tests, just logging them to console, otherwise letting them to pass.